src/Packetery/Module/Framework/WcAdapter.php

```diff
@@ -143,4 +145,13 @@ public function hasBlockInPage( int $page, string $blockName ): bool {
 	public function dataStoreLoad( string $objectType ): object {
 		return WC_Data_Store::load( $objectType );
 	}
+
+	/**
+	 * @param array<string, mixed> $args
+	 *
+	 * @return WC_Order[]|stdClass
```

Comment:
Poznámka na druhé kolo: rád bych zde viděl `WC_Abstract_Order[]` a `: array` - v případě `stdClass` by mělo stačit vytáhnout property `orders`.